restapi/endpoints_test.go

```diff
@@ -119,9 +127,13 @@ func (suite *endpointTestSuite) TestGetAllStories() {
 	response.Object().Value("payload").Array().Element(0).Object().Value("ID").Equal(1)
 	response.Object().Value("payload").Array().Element(0).Object().Value("content").Equal("Children")
 	response.Object().Value("payload").Array().Element(0).Object().Value("title").Equal("The Little Prince")
+	response.Object().Value("payload").Array().Element(0).Object().Value("summary").Equal("Summary")
+	response.Object().Value("payload").Array().Element(0).Object().Value("current_city").Equal("Toronto")
 	response.Object().Value("payload").Array().Element(1).Object().Value("ID").Equal(2)
 	response.Object().Value("payload").Array().Element(1).Object().Value("content").Equal("Fiction")
 	response.Object().Value("payload").Array().Element(1).Object().Value("title").Equal("Hitchhiker's Guide to the Galaxy")
+	response.Object().Value("payload").Array().Element(1).Object().Value("summary").Equal("Summary")
```

Comment:
super tiny thing, but it might be best to change the summary text to be slightly different? Even "summary1" and "summary2" would be good.